### PR TITLE
feat(solana): enable using address lookup tables

### PR DESCRIPTION
**Pull request by guibescos**
We already support address lookup tables in @pyth-network/solana-utils but we didn't expose them in the receiver sdk

### CODE REVIEW

File: target_chains/solana/sdk/js/solana_utils/src/transaction.ts [hunk 1]
```diff
@@ -161,11 +161,11 @@ export class TransactionBuilder {
   constructor(
     payer: PublicKey,
     connection: Connection,
-    accountLookupTable?: AddressLookupTableAccount
+    addressLookupTable?: AddressLookupTableAccount
   ) {
     this.payer = payer;
     this.connection = connection;
-    this.addressLookupTable = accountLookupTable;
+    this.addressLookupTable = addressLookupTable;
```
Comment (guibescos) [inline target_chains/solana/sdk/js/solana_utils/src/transaction.ts:168]: drive-by rename

**Review by ali-bahjati — APPROVED** [review]
